src/commands/node.mjs

```diff
@@ -385,6 +387,119 @@ export class NodeCommand extends BaseCommand {
     return true
   }
 
+  /**
+   * @param {string} namespace
+   * @param {string} nodeId
+   * @param {TaskWrapper} task
+   * @param {string} title
+   * @param {number} index
+   * @param {number} [status]
+   * @param {number} [maxAttempts]
+   * @param {number} [delay]
+   * @param {number} [timeout]
+   * @returns {Promise<string>}
+   */
+  async checkNetworkNodeActiveness (namespace, nodeId, task, title, index,
+    status = NodeStatusCodes.ACTIVE, maxAttempts = 120, delay = 1_000, timeout = 1_000) {
+    nodeId = nodeId.trim()
+    const podName = Templates.renderNetworkPodName(nodeId)
+    const podPort = 9_999
+    const localPort = 19_000 + index
+    task.title = `${title} - status ${chalk.yellow('STARTING')}, attempt ${chalk.blueBright(`0/${maxAttempts}`)}`
+
+    const srv = await this.k8.portForward(podName, localPort, podPort)
+
+    let attempt = 0
+    let success = false
+    while (attempt < maxAttempts) {
+      const controller = new AbortController()
+
+      const timeoutId = setTimeout(() => {
+        task.title = `${title} - status ${chalk.yellow('TIMEOUT')}, attempt ${chalk.blueBright(`${attempt}/${maxAttempts}`)}`
+        controller.abort()
+      }, timeout)
+
+      try {
+        const url = `http://${LOCAL_HOST}:${localPort}/metrics`
+        console.log(url)
+        const response = await fetch(url, { signal: controller.signal })
+
+        if (!response.ok) {
+          task.title = `${title} - status ${chalk.yellow('UNKNOWN')}, attempt ${chalk.blueBright(`${attempt}/${maxAttempts}`)}`
+          clearTimeout(timeoutId)
+          throw new Error()
+        }
+
+        const text = await response.text()
+        const statusLine = text
+          .split('\n')
+          .find(line => line.startsWith('platform_PlatformStatus'))
+
+        if (!statusLine) {
+          task.title = `${title} - status ${chalk.yellow('STARTING')}, attempt: ${chalk.blueBright(`${attempt}/${maxAttempts}`)}`
+          clearTimeout(timeoutId)
+          throw new Error()
```

Comment:
here you left the port forward open without closing it

Reply:
its in a try catch

Reply:
> here you left the port forward open without closing it

its in a try catch

Reply:
here you left the port forward open without closing it

its in a try catch

Reply:
instead of throwing a new Error why not just use continue, I think continue would be more clear for what you are using it for.